docs/iscsi-authentication.md

```diff
@@ -0,0 +1,71 @@
+This document describes how to associate a k8s secret with a VM for the purpose of iscsi initiator authentication.
+
+*NOTE: Only client authentication is supported at this time, meaning that the iscsi target with authenticate an initiator has permissions to access a device, but that a initiator can not authenticate the target.*
+
+## Workflow
+1. create a k8s secret containing the password and username fields.
+```
+cat << END > my-chap-secret.yaml
+apiVersion: v1
+kind: Secret
+metadata:
+  name: myK8sSecretID
+  namespace: default
+
```

Comment:
nah, i don't think that's necessary. The secret name can just be formatted however k8s allows it to be. usually '-' is the delimiter. I don't think we need to be explicit about it.